pipeline/extract.py

```diff
@@ -85,20 +87,72 @@ def get_website_from_url(url: str) -> str:
     return website_url
 
 
-def scrape_from_html(html_content: bytes, url: str, product_id: int) -> dict:
+def scrape_pricing_process(html_content: bytes, url: str, product_id: int) -> dict:
+    """ Chooses which scraper to use based off of the URL """
+
+    website_url = get_website_from_url(url)
+
+    if "https://store.steampowered.com/" in url:
+        return scrape_from_steam_html(html_content, url, product_id)
+
+    if "https://www.amazon.co" in url:
+        return scrape_from_amazon_html(html_content, url, product_id)
+
+    logging.error(
+        "Cannot scrape that URL, since it's not an Amazon/Steam webpage.")
+    return
+
+
+def scrape_from_amazon_html(html_content, url, product_id):
     """ Scrapes from html to get a dictionary with the:
     - Product_ID 
     - product_name
     - original_price
     - discount_price
     - website 
-    """
+    for Amazon Product Pages."""
+    s = BeautifulSoup(html_content, 'html.parser')
+
+    results = s.find("div", id="corePriceDisplay_desktop_feature_div")
+
+    if not results:
+        logging.error("Can't scrape from Amazon URL")
+        return None
+
+    game_title_elem = s.find(id="productTitle")
```

Comment:
Can this be renamed to something else - This variable name is specific to Steam products. Can rename it to product_name_element or something.